addons/google_calendar/models/calendar_recurrence_rule.py

```diff
@@ -136,8 +167,14 @@ def _create_from_google(self, gevents, vals_list):
             vals['calendar_event_ids'] = [(4, base_event.id)]
             # event_tz is written on event in Google but on recurrence in Odoo
             vals['event_tz'] = gevent.start.get('timeZone')
+            attendee_values[base_event.id] = {'attendee_ids': base_values.get('attendee_ids')}
+
         recurrence = super(RecurrenceRule, self.with_context(dont_notify=True))._create_from_google(gevents, vals_list)
-        recurrence.with_context(dont_notify=True)._apply_recurrence()
+        generic_values_creation = dict()
+        for rec in recurrence:
+            if attendee_values.get(rec.base_event_id.id):
+                generic_values_creation[rec.id] = attendee_values[rec.base_event_id.id]
```

Comment:
If you want you could do it like this:
```suggestion
        generic_values_creation = {
            rec.id: attendee_values[rec.base_event_id.id] 
            for rec in recurrence if attendee_values.get(rec.base_event_id.id)
        } 
```
🙂